src/osd/ReplicatedPG.h

```diff
@@ -916,8 +916,7 @@ class ReplicatedPG : public PG, public PGBackend::Listener {
   void hit_set_trim(RepGather *repop, unsigned max); ///< discard old HitSets
   void hit_set_in_memory_trim();                     ///< discard old in memory HitSets
 
-  hobject_t get_hit_set_current_object(utime_t stamp);
-  hobject_t get_hit_set_archive_object(utime_t start, utime_t end);
+  hobject_t get_hit_set_archive_object(utime_t start, utime_t end, bool using_gmt=true);
```

Comment:
I'd rather be explicit about using_gmt everywhere.  Probably better not to make that argument optional.